go/tools/macrobench/results.go

```diff
@@ -107,3 +109,30 @@ func GetResultsForLastDays(macroType MacroBenchmarkType, source string, lastDays
 	}
 	return macrodetails, nil
 }
+
+// insertToMySQL inserts the given MacroBenchmarkResult to MySQL using a *mysql.Client.
+// The MacroBenchmarkResults gets added in one of macrobenchmark's children tables.
+// Depending on the MacroBenchmarkType, the insert will be routed to a specific children table.
+// The children table QPS is also inserted.
+func (mbr *MacroBenchmarkResult) insertToMySQL(benchmarkType MacroBenchmarkType, macrobenchmarkID int, client *mysql.Client) error {
+	if client == nil {
+		return errors.New(mysql.ErrorClientConnectionNotInitialized)
+	} else if benchmarkType == "" {
+		return errors.New(IncorrectMacroBenchmarkType)
+	}
```

Comment:
nit: change remove else as the if condition has return.

Reply:
Sounds good, fixed them both in 0f13367.